server/plugin/plg_editor_onlyoffice/index.go

```diff
@@ -72,6 +73,21 @@ func init() {
 			return f
 		}).String()
 	}
+
+	can_edit = func() bool {
+	return Config.Get("features.office.can_edit").Schema(func(f *FormElement) *FormElement {
+		if f == nil {
+			f = &FormElement{}
+		}
+		f.Id = "onlyoffice_can_edit"
```

Comment:
The id should be in the "plugin_enable" Target attribute so it is only shown when only office is enabled